src/styles/List.scss

```diff
@@ -0,0 +1,42 @@
+/* Include the padding and border in an element's total width and height */
+* {
+    box-sizing: border-box;
+  }
+  /* Style the close button */
```

Comment:
Why have you commented this code?
I mean its obvious.

```css
.header {}
```

The above code styles header, I know, it's not a mandatory comment it, so it's a mandatory NOT COMMENT OBVIUS CODE. Please remove all unnecessary comments

Reply:
I haved remove all the unnecesary comments from the files.